Find a functionally equivalent code:android.graphics.Path_Delegate.lineTo:COMMENT
Method Modifier: private     
Comment:/**
 * Add a line from the last point to the specified point (x,y).
 * If no moveTo() call has been made for this contour, the first point is
 * automatically set to (0,0).
 *
 * @param x The x-coordinate of the end of a line
 * @param y The y-coordinate of the end of a line
 */

@@ -1,3 +1,6 @@
 {
+    if (isEmpty()) {
+        mPath.moveTo(mLastX = 0, mLastY = 0);
+    }
     mPath.lineTo(mLastX = x, mLastY = y);
 }

